packages/runtime/src/next/api-handler/handlers/proxy-preview-mode.ts

```diff
@@ -158,6 +158,6 @@ async function proxyPreviewModeApiRouteHandler(
 
   const arrayBuffer = await response.arrayBuffer()
 
-  res.write(arrayBuffer)
+  res.write(Buffer.from(arrayBuffer))
```

Comment:
Or maybe we should do `new Uint8Array` in case this ever runs in [Edge Runtime](https://nextjs.org/docs/app/api-reference/edge)?

Reply:
Yeah, good point. Is it just a matter if calling the constructor?

Reply:
Yes, `new Uint8Array(arrayBuffer)`. I think `Uint8Array` is just a [typed projection of `ArrayBuffer`](https://javascript.info/arraybuffer-binary-arrays). I'll switch over just in case.